src/App.tsx

```diff
@@ -1,25 +1,37 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
-import { getNumbers } from './utils';
-
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-const items = getNumbers(1, 42).map(n => `Item ${n}`);
+import { Pagination } from './components/Pagination';
 
 export const App: React.FC = () => {
+  const [currentPage, setCurrentPage] = useState(1);
+  const [itemsPerPage, setItemsPerPage] = useState(5);
+  const totalItems = 42;
+
+  const lastVisibleItem = currentPage * itemsPerPage;
+  const firstVisibleItem = lastVisibleItem - itemsPerPage;
```

Comment:
The calculation of 'lastVisibleItem' and 'firstVisibleItem' does not account for the case when the last page may have fewer items than 'itemsPerPage'. Consider adding logic to handle this edge case correctly.